cert-mgr-contour-fcd/install.md

```diff
@@ -116,27 +152,45 @@ To install cert-manager from the Tanzu Application Platform package repository:
       / Retrieving installation details for cert-manager...
       NAME:                    cert-manager
       PACKAGE-NAME:            cert-manager.tanzu.vmware.com
-      PACKAGE-VERSION:         1.5.3+tap.1
+      PACKAGE-VERSION:         2.0.0
       STATUS:                  Reconcile succeeded
-      CONDITIONS:              [{ReconcileSucceeded True  }]
+      CONDITIONS:              [{ReconcileSucceeded True}]
       USEFUL-ERROR-MESSAGE:
       ```
 
-      Verify that `STATUS` is `Reconcile succeeded`
+   Verify that `STATUS` is `Reconcile succeeded`
+
+6. Verify that cert-manager is up and running:
 
       ```console
-      kubectl get deployment cert-manager -n cert-manager
+      kubectl get deployment -n cert-manager
       ```
 
       For example:
 
       ```console
-      $ kubectl get deploy cert-manager -n cert-manager
-      NAME           READY   UP-TO-DATE   AVAILABLE   AGE
-      cert-manager   1/1     1            1           2m18s
+      $ kubectl get deployment -n cert-manager
+      NAME                      READY   UP-TO-DATE   AVAILABLE   AGE
+      cert-manager              1/1     1            1           5m
+      cert-manager-cainjector   1/1     1            1           5m
+      cert-manager-webhook      1/1     1            1           5m
       ```
 
-      Verify that `STATUS` is `Running`
+7. Verify that the self-signed `ClusterIssuer` is present:
+
+      ```console
+      kubectl get clusterissuer
```

Comment:
minot but do we want to add `-n cert-manager` ? (Unless the namespace of the clusterissuers are configurable in which case I'm sure the user can figure out what namespace to use)

Reply:
@Petahhh Great question! Since `ClusterIssuer` is a cluster-scoped resource providing the `-n/--namespace` option is not only redundant, it's probably misleading.